hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/ReconTaskControllerImpl.java

```diff
@@ -100,23 +100,23 @@ public void registerTask(ReconOmTask task) {
   @Override
   public synchronized void consumeOMEvents(OMUpdateEventBatch events, OMMetadataManager omMetadataManager) {
     if (!events.isEmpty()) {
-      Collection<NamedCallableTask<Pair<String, Pair<Integer, Boolean>>>> tasks = new ArrayList<>();
-      List<Pair<String, Integer>> failedTasks = new ArrayList<>();
+      Collection<NamedCallableTask<Pair<String, Pair<Map<String, Integer>, Boolean>>>> tasks = new ArrayList<>();
+      List<Pair<String, Map<String, Integer>>> failedTasks = new ArrayList<>();
       for (Map.Entry<String, ReconOmTask> taskEntry :
           reconOmTasks.entrySet()) {
         ReconOmTask task = taskEntry.getValue();
         ReconTaskStatusUpdater taskStatusUpdater = taskStatusUpdaterManager.getTaskStatusUpdater(task.getTaskName());
         taskStatusUpdater.recordRunStart();
         // events passed to process method is no longer filtered
-        tasks.add(new NamedCallableTask<>(task.getTaskName(), () -> task.process(events, 0)));
+        tasks.add(new NamedCallableTask<>(task.getTaskName(), () -> task.process(events, new HashMap<>())));
```

Comment:
it can be Collections.emptyMap() to avoid allocation. Similar can be done at different places where new of hashmap is done.

Reply:
Done.